Cython/Compiler/CmdLine.py

```diff
@@ -4,11 +4,17 @@
 
 from __future__ import absolute_import
 
+import sys
 import os
 from argparse import ArgumentParser, Action, SUPPRESS
 from . import Options
 
 
+if sys.version_info[:2] < (3, 3):
```

Comment:
```suggestion
if sys.version_info < (3, 3):
```

Tiny style comment - this works fine with tuples of different lengths